packages/edit-site/src/components/save-panel/index.js

```diff
@@ -11,6 +11,7 @@ import {
 	EntitiesSavedStates,
 	useEntitiesSavedStatesIsDirty,
 	privateApis,
+	EntitiesSavedStatesDialogWrapper,
```

Comment:
I think we should expose it as a private API initially, So, define it here:

https://github.com/WordPress/gutenberg/blob/961000076e74a6eea419d79ff8e16ae54a73463b/packages/editor/src/private-apis.js#L10